semeio/jobs/configs/STEA

```diff
@@ -0,0 +1,4 @@
+EXECUTABLE     ../scripts/fm_stea
```

Comment:
I tought I had a comment here, but the `../scripts/fm_stea` should be `../scripts/fm_stea.py`, right?

Reply:
right!